app/app.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const app = express();
+
+//init database
```

Comment:
why are you leaving these comments here?

Reply:
Removed